src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int MY_DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size = 0;
+    private int threshold;
+    private Node<K, V> [] table;
+
+    public MyHashMap() {
+        this.table = new Node[MY_DEFAULT_INITIAL_CAPACITY];
+        threshold = (int) (MY_DEFAULT_INITIAL_CAPACITY * LOAD_FACTOR);
+    }
+
+    class Node<K, V> {
```

Comment:
Common mistakes (jv-hash-map): Remember to parameterize the Node class. Class `Node` should take key and value, so we should parameterized it to prevent problems with data types.